css/index.css

```diff
@@ -1,3 +1,408 @@
+    margin-bottom: 15px;
+}
+
+body .toppage nav a {
+	color: grey;
+	text-decoration: none;
+	margin-right: 50px;
+}
+
+body .toppage nav a:hover {
+color:black;
+}
+
+.toppage {
+	display: flex;
+	align-items: baseline;
+	justify-content: center;
+}
+
+.container {
+	display: flex;
+	justify-content: center;
+	margin-top: 20px;
+	margin-bottom: 20px;
+}
+
+.services {
+	display: flex;
+	flex-wrap: wrap;
+	margin-top: 20px;
+	padding-bottom: 30px;
+	border-bottom: 1px solid grey;
+}
+
+#centered {
+	display: flex;
+	justify-content: center;
+}
+
+.middle6 {
+	display: flex;
+	flex-wrap: wrap;
+	width: 400px;
+	border: 1px solid grey;
+	padding: 20px;
+	background-color: #ECEBEB;
+	margin: 20px;
+}
+
+button {
+	margin-top: 20px;
+	background-color: white;
+	border: 1px solid grey;
+	padding: 5px;
+	padding-left: 25px;
+	padding-right: 25px;
+	
+}
+
+.middle6 button:hover {
+background-color: grey;
+}
+
+#mids {
+	justify-content: center;
+	display: flex;
+	flex-wrap: wrap;
+	padding-bottom: 20px;
+	border-bottom: 2px solid black;
+}
+
+.container-services {
+	width: 880px;
+	margin: 0 auto;
+}
+
+.servicefacts {
+	display: flex;
+	flex-wrap: wrap;
+	width: 400px;
+	flex-direction: column;
+	margin-left: 20px;
+}
+
+.bottom3 {
+	margin-top: 40px;
+	display: flex;
+	justify-content: center;
+}
+
+.thelist li {
+	display: flex;
+	flex-wrap: wrap;
+	margin: 10px;
+	list-style-type: circle;
+}
+
+.thelist {
+	margin-top: 20px;
+	
+}
+
+.servicefacts h3 {
+	margin: 0px;
+display: flex;
+height: 25px;
+margin-top: 0px;
+}
+
+.spacee {
+	margin-right: 25px;
+}
+
+.bottom {
+	display: flex;
+	justify-content: center;
+	margin-top: 70px;
+	margin-bottom: 30px;
+}
+
+.h20 {
+	font-size: 20px;
+}
+
+.servicetitle {
+	margin-top: 5px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.head nav a {
+	color: grey;
+	text-decoration: none;
+	margin-right: 50px;
+}
+
+.head {
+	display: flex;
+	align-items: baseline;
+	justify-content: center;
+}
+
+
+.innovation {
+	font-size: 70px;
+	text-align: center;
+}
+
+.lefts {
+display: flex;
+justify-content: center;
+flex-direction: column;
+align-items: center;
+margin-left: 50px;
+}
+
+.main {
+	display: flex;
+	margin-top: 60px;
+	margin-bottom: 30px;
+	border-bottom: 1px solid grey;
+	padding-bottom: 50px;
+	justify-content: space-between;
+}
+
+.codeimg {
+
+}
+
+.startedd {
+	text-align: center;
+}
+
+.navigation {
+	margin-right: 30px;
+}
+
+.firstmid {
+	display: flex;
+	justify-content: space-between;
+	margin-bottom: 40px;
+}
+
+.seperator {
+	display: flex;
+	flex-wrap: wrap;
+	width: 400px;
+
+}
+
+.threemid {
+display: flex;
+flex-wrap: wrap;
+width: 250px;
+}
+
+.thirdmid {
+	display: flex;
+	justify-content: space-between;
+	margin-top: 30px;
+	border-bottom: 1px solid grey;
+	padding-bottom: 30px;
+}
+
+.foot {
+	margin-top: 30px;
+width: 250px;
+}
+
+.cpyr {
+	display: flex;
+	justify-content: center;
+	margin-top: 30px;
+	margin-bottom: 40px;
+}
+
```

Comment:
I recommend adding comments to your code to make it more organized / easier to read for your future self and other developers.